enatega-multivendor-app/src/screens/Reorder/Reorder.js

```diff
@@ -153,13 +153,16 @@ function Reorder(props) {
         })
       })
     }
+    const populateAddons = addons.filter((addon) =>
+      food.variations[0].addons.includes(addon._id)
```

Comment:
food variation addons can be undefined, in this case it will break using include, apply check or use chaining operator here as well